README.md

```diff
@@ -5,29 +5,33 @@
 
 The Rust-ML working group is currently at an early stage of development, and is still deciding on several choices at an organizational level. This document will be updated accordingly as those decisions are made.
```

Comment:
For me the `several choices` is a bit too vague. Perhaps we can word this as 

> The Rust-ML working group is currently at an early stage of development, and its prospect role and function in the wider Rust ecosystem is currently explored. This document will reflect those updates and inform the wider community on decisions made by the Rust-ML group.

I think it is useful to add a general statement, why we believe that Rust and Machine Learning is worthwhile:
> We strongly believe that Rust can contribute to the Machine Learning community with its correctness and speed. It brings the promise of a unified scientific workhorse by combining elegance and performance, previously only found in separate languages. We hope that this will produce libraries, which are easier to use, easier to iterate on and give a standardized level of performance across the ecosystem.

Reply:
:+1: I really like these changes

Reply:
Me too! I made a couple of edits (mostly grammar) to try to make it flow a little more, but hopefully it still communicates what you wanted. If not, we can change it back